entries/callbacks.fireWith.xml

```diff
@@ -3,11 +3,11 @@
   <title>callbacks.fireWith()</title>
   <signature>
     <added>1.7</added>
-    <argument name="context" optional="true">
+    <argument name="context" optional="true" type="Anything">
       <desc>A reference to the context in which the callbacks in the list should be fired.</desc>
     </argument>
-    <argument name="args" optional="true">
-      <desc>An argument, or array of arguments, to pass to the callbacks in the list.</desc>
+    <argument name="args" optional="true" type="Array">
+      <desc>An array of arguments to pass to the callbacks in the list. If omitted or undefined, no arguments will be passed.</desc>
```

Comment:
Hey @dratini0 , thanks for the PR! I don't think this change is correct, though. Looks to me like `args` can be "Anything" — an argument or array of arguments like the doc originally stated.

```
args = args || [];
args = [ context, args.slice ? args.slice() : args ];
queue.push( args );
```
https://github.com/jquery/jquery/blob/3e902a812014e8a6980e08599c4840b1cb969f7c/src/callbacks.js#L208-L210

Reply:
Take a look at the output of this fiddle then: https://jsfiddle.net/48Lmdmmf/

I have missed line 208, and assumed this was the intended behaviour. However, now I see that we are looking at a bug in jQuery, not in the documentation. I will put a pull request through shortly for this:

```js
args = [ context, args.slice ? args.slice() : [args] ];
```

I will also update this PR accordingly.